src/styles/index.ts

```diff
@@ -5168,10 +5170,11 @@ const styles = (theme: ThemeColors) =>
 
         emptyStateFolderWebStyles: {
             ...sizing.w100,
-            minWidth: 300,
+            minWidth: 400,
```

Comment:
@Krishna2323 Friendly reminder that you need to make sure you test this one on smaller device like iPhone8

Reply:
Thanks man! Appreciate it, I'll give it a final review today!